.github/workflows/release.yml

```diff
@@ -35,6 +35,17 @@ jobs:
 
       - run: echo ("VERSION_NAME=" + $env:GITHUB_REF_NAME.TrimStart("v")) >> $env:GITHUB_ENV
       - run: echo $VERSION_NAME
+      
+      - name: Release to Winget
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: jsdelivr.Globalping
+          installers-regex: 'Windows'
```

Comment:
Also yeah I'm not sure how it reads the directories and if we need a path or not

Reply:
That makes sense! I agree the regex might not be clear since it may not recursively check directories. One thing we could do is run `cd dist` before this command to resolve that.

Reply:
Any chance you can test it on a fork? I think you already looked into it previously

Reply:
I'm currently travelling for the next 2 weeks so that may be a little difficult to test on my phone 😅 

I do think we'll need to follow through on the `cd dist` suggestion though.

Reply:
Please take a look when you have time